Directory.Build.targets

```diff
@@ -22,4 +22,20 @@
     <NETCoreAppMaximumVersion>$(MajorVersion).$(MinorVersion)</NETCoreAppMaximumVersion>
   </PropertyGroup>
 
+  <Target Name="AddNetCoreAppCurrentToKnownFrameworkReference"
```

Comment:
Does this have to be a target?  Can it be just an ItemGroup?

Reply:
We dont have value of ```BundledNETCoreAppPackageVersion``` at evaluation time. It is generated on the fly.